#!/bin/sh /usr/share/dpatch/dpatch-run

## Makefile-fPIC-fix patch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adds -fPIC to Makefile to fix potential FTBFS.

@DPATCH@
diff -urNad /usr/src/cvs/vdrdevel-plugin-rotor/Makefile vdrdevel-plugin-rotor/Makefile
--- /usr/src/cvs/vdrdevel-plugin-rotor/Makefile	Sun May 16 15:48:12 2004
+++ vdrdevel-plugin-rotor/Makefile	Sat Sep 11 00:28:18 2004
@@ -18,7 +18,7 @@
 ### The C++ compiler and options:
 
 CXX      ?= g++
-CXXFLAGS ?= -O2 -Wall -Woverloaded-virtual
+CXXFLAGS ?= -fPIC -O2 -Wall -Woverloaded-virtual
 
 ### The directory environment:
 
